order-service/src/main/resources/db/changelog/v1/v1-accumulative-changelog.yml

```diff
@@ -1,3 +1,3 @@
 databaseChangeLog:
   - include:
```

Comment:
where is the Kafka implementation for orderService which is required by requirements